src/lib.rs

```diff
@@ -709,6 +710,19 @@ mod tests {
         assert_eq!(dispatcher.params[0], &[i64::MAX as i64]);
     }
 
+    #[test]
+    fn reset_param_on_clear() {
+        static INPUT: &[u8] = b"\x1bP1X\x9c\x1b[31mH";
+        let mut dispatcher = CsiDispatcher::default();
```

Comment:
Isn't it a bit inconsistent to use 7-bit escapes for the DCS start and CSI, but an 8-bit escape for ST

Reply:
For a test it's both the same, but it's easier to understand `\x9c` than `\x1b\\` imho. I'm quite used to seeing `\x1b[` or `\x1b]`, but I just can't quite get used to `\x1b\\`, it might be because it requires `\` to be escaped?

Reply:
Good enough for me :)